code/datums/elements/nerfed_pulling.dm

```diff
@@ -0,0 +1,52 @@
+/// This living will be slower when pulling/moving some dangerous objects in its obj_flags
+/datum/element/nerfed_pulling
+	element_flags = ELEMENT_DETACH
+
+	var/static/list/nerf_pulling_of_typecache = typecacheof(list(
```

Comment:
Gonna turn this into an argument for a bespoke element, rather than an arbitrary static.